src/scripts/run-td-extract.sh

```diff
@@ -96,7 +103,9 @@ if [[ -n "${SKIP_SQL_SCRIPTS}" ]]; then
     args+=( --skip-sql-scripts "${SKIP_SQL_SCRIPTS}" )
 fi
 
+args+=($@)
+
 CLASSPATH="$(dirname "$0")/ExtractionTool_deploy.jar:${TERAJDBC4_JAR}"
-java -cp "${CLASSPATH}" \
+echo java -cp "${CLASSPATH}" \
```

Comment:
This will cause the command to be printed out only, no?

Reply:
Yes, good catch. This needs to be reverted.